web/viewWorkingHoursResultsReport.php

```diff
@@ -145,6 +146,10 @@ function populate() {
                     var index = store.findExact('login', selectedLogin);
                     grid.getSelectionModel().selectRow(index);
                     grid.getView().focusRow(index);
+                } else {
+                    loggedInUserIndex = store.findExact('login', loggedInUser);
+                    grid.getSelectionModel().selectRow(loggedInUserIndex);
+                    grid.getView().focusRow(loggedInUserIndex);
```

Comment:
Why create an else branch with almost the same code, when you could set `selectedLogin = loggedInUser` by default :)